Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -728,12 +731,317 @@ DECLARE_SOA_COLUMN(TimeZNC, timeZNC, float);                    //!
 DECLARE_SOA_COLUMN(TimeZPA, timeZPA, float);                    //!
 DECLARE_SOA_COLUMN(TimeZPC, timeZPC, float);                    //!
 } // namespace zdc
+namespace zdc_001 // Revised table, required for dynamic in-place replacements
```

Comment:
Please use the same namespace, the versioning should only be done at the table level, otherwise you will have to rename the namespace later.

Reply:
Hi Anton, as explained in the comment inside the code, I need to have different namespaces to have different types of columns with the same names: we are swapping out a regular column with a dynamic column in the new version of the table, and these have the same exact name. This cannot be done keeping the same namespace, unfortunately...

Reply:
Since the columns will have different type, they may as well have a different name. In fact, you can rename old columns (only the type names, not the getters). This will be transparent for the users, since they are only using the getters.

Reply:
The latest commit should take care: it uses only the `zdc` namespace but changes the column _names_ for the dynamic columns while keeping the same _getters_. Can you please take a look @aalkin ? Thanks a lot!